packages/big-design/src/components/List/Item/styled.tsx

```diff
@@ -69,6 +69,19 @@ export const StyledListItem = styled.li<ListItemProps<unknown>>`
   }
 `;
 
+export const StyledLabel = styled.span<any>`
+  margin-right: ${({ theme, isSelected }) => (isSelected ? theme.helpers.remCalc(8) : theme.helpers.remCalc(28))};
+
+  &:after {
+    display: block;
+    content: attr(content);
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+    font-weight: 700;
+  }
```

Comment:
Can you help me understand why we need this?

Reply:
List with items should have static width according to the ticket. List width equals to the width of the biggest item. When some item activates text becomes bold and wider, so we should calculate bold text to define item width. Styles above add pseudo element with bold text. I has no height but its width help us to reserve space needed for bold text when item is active.

Reply:
Ok, thanks for the explanation!

Do you think we can use something like `content: attr(textContent)` to remove the redundant `content` prop?